test/test_helper.rb

```diff
@@ -19,6 +19,28 @@ def teardown
   def create_chip(name, price, description)
     Chip.create(name: name, price: price, description: description)
   end
+
+  def create_shop_and_logged_in_user
+    category_1 = Oil.create(name: "Lard")
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
+    item_2 = Chip.create(name: "Old Chips", price: 20, description: "Super yummy", oil_id: category_1.id, status: "retired")
+    user = User.create(username: "John", password: "Password")
+    order = user.orders.create(total_price: 20)
+    order.chip_orders.create(chip_id: item_1.id, quantity: 1, subtotal: 20)
+    order.chip_orders.create(chip_id: item_2.id, quantity: 1, subtotal: 20)
+
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.